src/utils/cryptography/interface.cairo

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.10.0 (utils/cryptography/interface.cairo)
+
+use starknet::ContractAddress;
+
+#[starknet::interface]
+trait INonces<TState> {
+    fn nonces(self: @TState, owner: ContractAddress) -> felt252;
```

Comment:
i think felt252 might be just fine. but I wonder if you had any thoughts on using u256

Reply:
felt252 should be cheaper than u256 even in the future if we have a smaller felt. As Ariel confirmed, I don't think fel252 will be removed, so for this case, it suits us better IMO.